media/css/firefox/new/scene2_email.scss

```diff
@@ -41,6 +41,24 @@ $image-path: '/media/protocol/img';
     p {
         margin: 0;
     }
+
+    .windows-xpvista {
+        display: none;
+        color: $color-black;
+        background-color: #FAFAD2;
```

Comment:
This isn't one of our [brand colors](https://protocol.mozilla.org/fundamentals/color.html). I'd suggest changing it to `$brand-lemon-yellow` and allowing the warning to run full width of the screen:

![screenshot_2018-09-13 download firefox free web browser 1](https://user-images.githubusercontent.com/400117/45492106-5f1a3b00-b763-11e8-9c12-62015e8ddc48.png)